.circleci/config.yml

```diff
@@ -161,10 +161,7 @@ jobs:
         name: Docker Build
         # Run MYSQL image and build v2xhub docker image
         command: |
-          cd configuration/amd64/
-          docker-compose up -d db
-          cd ../../
-          docker build --network=host -t usdotfhwaops/v2xhubamd:${CIRCLE_BRANCH,,} .
+          docker build -t usdotfhwaops/v2xhubamd:${CIRCLE_BRANCH,,} .
```

Comment:
Is the docker-compose command no longer needed?  With us having have to handle ARM also now, I would think it's needed.

Reply:
No I have removed the dependency on mysql during the build process since we have moved the plugin install calls into the run script. I believe this is a better way to do this as well since it allows the build to happen independently or mysql and there are similar calls in the run script anyway.